bfdriver/efi/src/arch/intel_x64/efi_base/base.c

```diff
@@ -0,0 +1,93 @@
+
+#include "base.h"
+#include "boot.h"
+#include "mp_service.h"
+#include "bfelf_loader.h"
+#include "common.h"
+
+
+EFI_MP_SERVICES_PROTOCOL *g_mp_services;
+extern char target_vmm_start[];
+extern char target_vmm_end[];
+
+
+VOID __attribute__((ms_abi)) bf_start_hypervisor_on_core(VOID *data)
+{
+    (void)data;
+    _set_ne();
+    Print(L"Starting hypervisor on core.\n");
+    platform_start_core();
+
+    return;
+}
+
+boot_ret_t base_start_fn()
+{
+    EFI_STATUS status;
+
+    uint64_t target_vmm_size = (uint64_t)(target_vmm_end - target_vmm_start);
```

Comment:
Is target_vmm_end pointing to the byte directly after the last valid byte?

Reply:
I just checked, and the symbol it generates for that does appear to point one byte after the last byte of data

Reply:
:+1: